src/main/drivers/pwm_output.c

```diff
@@ -403,11 +432,61 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t motorCount, uint8_t command)
             }
 
             pwmCompleteDshotMotorUpdate(0);
-            delay(1);
+        }
+        delayMicroseconds(timeDelayUs);
+    } else {
+        for (uint8_t i = 0; i < motorCount; i++) {
+            if ((i == index) || (index == ALL_MOTORS)) {
+                if (dshotCommandControl.command[i] == 0) {
+                    dshotCommandControl.command[i] = command;
+                    dshotCommandControl.repeats = repeats;
```

Comment:
these fields will be overwritten even when command to other motor is pending .. is that safe?

Reply:
The trouble I think you could get into is if you were trying to send a repeat command like reverse motor on one motor at the same time as a beep command to another motor.  Might be able to get around that edge case by only allowing the command to be set if `dshotCommandControl.repeats == repeats || dshotCommandControl.repeats == 0`  but I'm trying to think if there is edge cases to that as well...

Reply:
I think the easiest way out of this will be to make this call fail if `pwmIsProcessingDshotCommand()` - having the restriction of being able to send commands to either 1 or all motors concurrently is not that bad.